### ISSUE TITLE
Failed to convert spans to metrics.

### ISSUE THREAD

**Issue by gou4shi1** [uptrace/uptrace#143]
The error logs:
```
2022-12-28T11:20:27.143Z	info	uptrace/main.go:111	starting Uptrace...	{"version": "v1.2.4", "config": "/etc/uptrace/uptrace.yml"}
2022-12-28T11:20:27.160Z	info	tracing/span_processor.go:47	starting processing spans...	{"threads": 8, "batch_size": 8000, "buffer_size": 128000}
2022-12-28T11:20:27.163Z	info	metrics/measure_processor.go:64	starting processing metrics...	{"threads": 8, "batch_size": 8000, "buffer_size": 128000}
[ch]  11:20:27.384   CREATE VIEW         108.458ms  CREATE MATERIALIZED VIEW "metrics_uptrace_tracing_spans_duration_mv" ON CLUSTER "default_cluster" TO measure_minutes AS SELECT s.project_id, 'uptrace.tracing.spans_duration' AS metric, toStartOfMinute(s.time) AS time, 'histogram' AS instrument, xxHash64(arrayStringConcat([toString(s."system"), toString(s."group_id"), toString(s."service_name"), toString(s."host_name"), toString(s."status_code")], '-')) AS attrs_hash, ['system', 'group_id', 'service', 'host', 'status'] AS attr_keys, [toString(s."system"), toString(s."group_id"), toString(s."service_name"), toString(s."host_name"), toString(s."status_code")] AS attr_values, toJSONString(map('span.name', toString(any(s."name")))) AS annotations, count() AS count, sum(s."duration" / 1000) AS sum, quantilesBFloat16State(0.5)(toFloat32(s."duration" / 1000)) AS histogram FROM spans_index AS s WHERE (NOT toFloat64OrDefault(s.system IN ('log:trace', 'log:debug', 'log:info', 'log:warn', 'log:error', 'log:fatal', 'log:panic', 'exceptions', 'other-events')) = 1) GROUP BY s.project_id, toStartOfMinute(s.time), toString(s."system"), toString(s."group_id"), toString(s."service_name"), toString(s."host_name"), toString(s."status_code") 	  *ch.Error: DB::Exception: There was an error on [9.0.16.13:9000]: Code: 60. DB::Exception: Table default.spans_index doesn't exist. (UNKNOWN_TABLE) (version 22.3.10.22) 
2022-12-28T11:20:27.384Z	error	metrics/init.go:38	initSpanMetrics failed	{"error": "createSpanMetric \"uptrace.tracing.spans_duration\" failed: createMatView failed: DB::Exception: There was an error on [9.0.16.13:9000]: Code: 60. DB::Exception: Table default.spans_index doesn't exist. (UNKNOWN_TABLE) (version 22.3.10.22)"}
github.com/uptrace/uptrace/pkg/metrics.Init
	github.com/uptrace/uptrace/pkg/metrics/init.go:38
main.glob..func2
	github.com/uptrace/uptrace/cmd/uptrace/main.go:139
github.com/urfave/cli/v2.(*Command).Run
	github.com/urfave/cli/v2@v2.23.5/command.go:271
github.com/urfave/cli/v2.(*Command).Run
	github.com/urfave/cli/v2@v2.23.5/command.go:264
github.com/urfave/cli/v2.(*App).RunContext
	github.com/urfave/cli/v2@v2.23.5/app.go:329
github.com/urfave/cli/v2.(*App).Run
	github.com/urfave/cli/v2@v2.23.5/app.go:306
main.main
	github.com/uptrace/uptrace/cmd/uptrace/main.go:68
runtime.main
	runtime/proc.go:250
2022-12-28T11:20:27.384Z	info	uptrace/main.go:351	starting monitoring metrics...	{"rules": 0}
```
The `metrics_from_spans` config is same as https://github.com/uptrace/uptrace/blob/master/config/uptrace.yml#L67

**Comment by gou4shi1** [comment]
The `uptrace` clickhouse table is created by `uptrace` and there is only `uptrace.spans_index` but not `default.spans_index`.

**Comment by vmihailenco** [comment]
Should be fixed in v1.2.5
